netci.groovy

```diff
@@ -44,7 +44,6 @@ def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolder
                     }
 
                     shell("unpacker LocalPackages.pack .")
-                    shell("chmod +x BuildCliComponents.sh")
```

Comment:
@brianrob No, as I changed the file attribute to executable in this PR.

---
In reply to: [100207823](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100207823) [](ancestors = 100207823)